liberty-maven-plugin/src/it/tests/skip-start-server-it/src/test/java/net/wasdev/wlp/maven/test/app/skipServerTest.java

```diff
@@ -0,0 +1,44 @@
+package net.wasdev.wlp.maven.test.app;
```

Comment:
Rename test to SkipServerTestIT.java so that it runs as a FailSafe test by default.  Note upper case S and TestIT